content/guides/user-guides/multi-cloud/object-archival-restore/_index.md

```diff
@@ -19,7 +19,7 @@ This guide shows how to archive objects into Cloud object storage and restore th
 To refer the API specs, please check [Cloud Object Archival API Specs](https://github.com/sodafoundation/multi-cloud/blob/master/openapi-spec/swagger.yaml)
 
 {{% notice note %}}
- Currently this feature is available only for AWS cloud
+ Currently this feature is available only for AWS, Azure and GCP cloud as backend and SONY-ODA as on-premise Cold storage
 {{% /notice %}}
 
 ## Installing SODA multi-cloud
```

Comment:
Could you please check the links at the Pre-requisite section below: they are not appearing correctly.
I believe the correct linking would be [How to register a Cloud Backend](/#how-to-register-a-cloud-backend)
`[How to register a Cloud Backend](/#how-to-register-a-cloud-backend)`
![image](https://user-images.githubusercontent.com/19162717/113402007-fac33380-93c1-11eb-84c9-7eb42633277b.png)

Reply:
Corrected. Thanks for checking